src/main/java/org/icgc/argo/program_service/services/ProgramService.java

```diff
@@ -314,4 +314,37 @@ public void removeProgram(String name) throws EmptyResultDataAccessException {
                 .listAll());
     return List.copyOf(new LinkedHashSet<ProgramEntity>(programs));
   }
+
+  public List<CancerEntity> listCancers() {
+    val cancers = cancerRepository.findAll();
+    return List.copyOf(new LinkedHashSet<CancerEntity>(cancers));
```

Comment:
Can it be simplified to ```
return cancerRepository.findAll();
``` ?

Reply:
`List.copyOf`will make it immutable which is generally a good thing.

You can simplify the whole method to:

```java
  public List<CancerEntity> listCancers() {
    return List.copyOf(cancerRepository.findAll());
  }
```